packages/connection/src/class-error-handler.php

```diff
@@ -296,18 +296,49 @@ public function wp_error_to_array( \WP_Error $error ) {
 		$user_id = $this->get_user_id_from_token( $data['token'] );
 
 		$error_array = array(
-			'error_code'    => $error->get_error_code(),
-			'user_id'       => $user_id,
-			'error_message' => $error->get_error_message(),
-			'error_data'    => $data,
-			'timestamp'     => time(),
-			'nonce'         => wp_generate_password( 10, false ),
+			'error_code'              => $error->get_error_code(),
+			'user_id'                 => $user_id,
+			'error_message'           => $error->get_error_message(),
+			'error_data'              => $data,
+			'timestamp'               => time(),
+			'nonce'                   => wp_generate_password( 10, false ),
+			'lost_active_master_user' => $this->track_lost_active_master_user( $error->get_error_code(), $data['token'], $user_id ),
```

Comment:
I don't think we need to create a new entry for this value. If our goal is to modify the error_message. Lets do so by directly modifying the error_message instead of a new flag that WP.com will use to overwrite the error_message sent. It'll be a cleaner approach.

Reply:
Of course!